torch_tvm/custom_tvm_ops/topi/contrib/quantize.cc

```diff
@@ -0,0 +1,78 @@
+#include <dmlc/logging.h>
+#include <tvm/runtime/registry.h>
+#include <tvm/runtime/util.h>
+#include <cmath>
+
+namespace tvm {
+namespace contrib {
+
+using namespace runtime;
+
+TVM_REGISTER_GLOBAL("tvm.contrib.find_minmax")
+    .set_body([](TVMArgs args, TVMRetValue* ret) {
+      DLTensor* input = args[0];
+      DLTensor* data_min = args[1];
+      DLTensor* data_max = args[2];
+      // calculate the data_min and data_max
+      auto data_ptr = static_cast<float *>(input->data);
```

Comment:
DLTensor has strides, should we assert those aren't used before doing this contiguous sweep?

Reply:
@bwasti, for minmax it should not matter though, right? Order in which elements are traversed is not important.